src/model/kaniOutputParser.ts

```diff
@@ -101,9 +101,20 @@ function parseChecksArray(checksArray: Array<string>): KaniResponse {
 	for (let i = 0; i < checksArray.length; i++) {
 		const checkInstance: string[] = checksArray[i].split('\n');
 		const checkInstanceObject: CheckInstance = convertChecktoObject(checkInstance);
-		if (checkInstanceObject.status == 'FAILURE') {
+		if (
+			checkInstanceObject.status == 'FAILURE' ||
+			checkInstanceObject.status == 'UNDETERMINED' ||
+			checkInstanceObject.status == 'UNREACHABLE' ||
+			checkInstanceObject.status == 'SATISFIED' ||
+			checkInstanceObject.status == 'UNSATISFIABLE'
+		) {
```

Comment:
nit, IMHO this is neater:
```suggestion
                const error_statuses = [                          
                    'FAILURE', 'UNDETERMINED', 'UNREACHABLE', 'SATISFIED', 'UNSATISFIABLE'                                 
                ];
		if (
		    error_statuses.includes(checkInstanceObject.status)
		) {
```

Reply:
Fixed. Thank you!